classes/decorator_trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator_base'
+
+class DecoratorTrimmer < DecoratorBase
+  def correct_name
+    @nameable.correct_name.slice(0..9)
+  end
+end
```

Comment:
- [OPTIONAL] You can also improve this code by checking if the name has a maximum of 10 characters before slice it and if it's longer it should trim the word. eg:


 ```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```